conda/environments/cudf_dev_cuda11.5.yml

```diff
@@ -20,7 +20,7 @@ dependencies:
   - python>=3.8,<3.10
   - numba>=0.54
   - numpy
-  - pandas>=1.0,<1.5.0dev0
+  - pandas=1.5.0rc0
```

Comment:
The pandas rcs will usually be available in on the `conda-forge/label/pandas_rc` channel so I think that channel will need to be added above.

Reply:
Added a temporary command to inject the channel.